model-deployment/containers/nim/README-MODEL-CATALOG.md

```diff
@@ -0,0 +1,75 @@
+# Overview
+
+Utilising Model Catalog to store Models in OCI. We describe two ways to achieve this: 
+
+* Storing zipped model file in Model Catalog
+* Utilising Model-by-reference Feature to download model
```

Comment:
We cant call it model by reference. External customers cant understand this.
Use Object storage to store the model and creating a model catalog pointing to Object storage bucket